src/components/calendar-views/ScheduleCalendarView.tsx

```
@@ -0,0 +1,274 @@
+                              <div className="flex items-center gap-1">
+                                <Clock className="h-3 w-3" />
+                                <span>All day</span>
+                              </div>
+                            )}
+                          </div>
+                          
+                          {/* Description preview (truncated) */}
+                          {event.description && (
+                            <div 
+                              className="text-xs text-muted-foreground line-clamp-2 mt-1"
+                              dangerouslySetInnerHTML={{ 
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)